exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/store/LocalPersistentStore.java

```diff
@@ -112,23 +127,65 @@ public static DrillFileSystem getFileSystem(DrillConfig config, Path root) throw
 
   @Override
   public Iterator<Map.Entry<String, V>> getRange(int skip, int take) {
+    //Marking currently seen modification time
+    long currBasePathModified = 0L;
+    try {
+      currBasePathModified = fs.getFileStatus(basePath).getModificationTime();
+    } catch (IOException ioexcp) {
+      ioexcp.printStackTrace();
```

Comment:
Please do not use `printStackTrace()`

Reply:
Will publish a log message and return an empty iterator for now. Not sure how to bubble up an error to the UI. I'll take a look at how we do so for profile deserialization as a guide